src/Compilers/VisualBasic/Portable/Parser/ParserFeature.vb

```diff
@@ -41,15 +41,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         UnconstrainedTypeParameterInConditional
         CommentsAfterLineContinuation
         InitOnlySettersUsage
-        CallerArgumentExpression
```

Comment:
>CallerArgumentExpression [](http://example.com/codeflow?start=8&length=24)

We are not doing language version checks on VB side?

Reply:
@AlekseyTs Not in C# as well.